src/ui/Form/DateField.tsx

```diff
@@ -47,28 +48,32 @@ function DatePicker<FormValues extends FieldValues>({
         // onDayChange won't work with user keyboard's input
         dayPickerProps={{ ...dayPickerProps, onDayClick: field.onChange }}
         placeholder=" "
-        formatDate={formatDate}
-        parseDate={parseDate}
+        formatDate={(date, format) => dayjs(date).format(format)}
```

Comment:
can you define these as functions first and call them here?

Reply:
It's just that defining them someone, I had to type the type annotations while no type annotation is needed if I defined it in place lol

Reply:
Thought you're not a fan of verbose component props as well. Still, we don't define onClick inline so keep the consistency.

Reply:
I am not but people change at 3:30 am. Making the change